novuhq/website
# feat: new sections for home page

This pull request brings new sections for home page:

- `used by` logos
- social reviews
- certificates for footer

**Steps to check:**

- open [preview](https://deploy-preview-105--novu-website.netlify.app/)
- make sure that everything looks and feels as expected
